moshi-sealed/codegen/src/main/kotlin/dev/zacsweers/moshix/sealed/codegen/ksp/MoshiSealedSymbolProcessorProvider.kt

```diff
@@ -275,9 +275,8 @@ private class MoshiSealedSymbolProcessor(environment: SymbolProcessorEnvironment
         }
       }
 
-    val ksFile = preparedAdapter.spec.originatingKSFiles().single()
     preparedAdapter.spec.writeTo(codeGenerator, aggregating = true)
-    preparedAdapter.proguardConfig?.writeTo(codeGenerator, ksFile)
+    preparedAdapter.proguardConfig?.writeTo(codeGenerator, type.containingFile)
```

Comment:
I initially implemented it that way but the generated proguard file references only the base type, so I'm wondering if it's necessary to depend on the sub types also.

It seems like the proguard file should be regenerated / processed, iff the base type is changed 🤔.

Reply:
hmmmm, I suppose you're right. That said, I'm curious though why `type.containingFile` is functionally different than fetching a single originating file? Looking at this, it seems it should return the same value? If not, I think this may be papering over a different bug

Reply:
gotcha, ok. Thanks!